lib/webmock/http_lib_adapters/curb_adapter.rb

```diff
@@ -333,6 +335,10 @@ def on_#{callback} &block
         METHOD
       end
 
+      def reset_webmock_method
+        @webmock_method = :get
```

Comment:
why :get and not nil?

Reply:
I think putting user in a `Pit of success` is always better. Since we all assume default method would be `:get` if we not specify it otherwise anyway.